mappings/net/minecraft/entity/ai/pathing/EntityNavigation.mapping

```diff
@@ -25,7 +25,7 @@ CLASS net/minecraft/class_1408 net/minecraft/entity/ai/pathing/EntityNavigation
 	METHOD <init> (Lnet/minecraft/class_1308;Lnet/minecraft/class_1937;)V
 		ARG 1 mob
 		ARG 2 world
-	METHOD method_18053 onBlockChanged (Lnet/minecraft/class_2338;)Z
+	METHOD method_18053 shouldRecalculatePath (Lnet/minecraft/class_2338;)Z
```

Comment:
Mind rename `shouldRecalculate` (`field_6679`) to `inRecalculateCooldown` or something like that? That field seems to be a marker that a recalculation is requested, yet it is still waiting because the 20 tick time delay in `RECAUCULATE_COOLDOWN` hasn't run out yet. Otherwise, this method looks confusing as it checks that field, which now looks like `if (shouldRecalculate) return false;`